src/Component/PreviewMap/PreviewMap.example.md

```diff
@@ -0,0 +1,43 @@
+This demonstrates the use of `Preview`.
+
+```jsx
+import * as React from 'react';
+import { PreviewMap } from 'geostyler';
+
+class PreviewMapExample extends React.Component {
```

Comment:
Not blocking, but I would love to see the example as a `FunctionComponent`.

Reply:
I'm with you on this point, but i skip this for now.